engine/inc/com/centreon/engine/commands/otel_interface.hh

```diff
@@ -67,17 +67,17 @@ class host_serv_list {
   void unregister_host_serv(const std::string& host,
                             const std::string& service_description);
 
-  bool is_allowed(const std::string& host,
-                  const std::string& service_description) const;
+  bool contains(const std::string& host,
+                const std::string& service_description) const;
 
   template <typename host_set, typename service_set>
-  host_serv_metric is_allowed(const host_set& hosts,
-                              const service_set& services) const;
+  host_serv_metric contains(const host_set& hosts,
+                            const service_set& services) const;
 };
 
 template <typename host_set, typename service_set>
-host_serv_metric host_serv_list::is_allowed(const host_set& hosts,
-                                            const service_set& services) const {
+host_serv_metric host_serv_list::contains(const host_set& hosts,
```

Comment:
contains is not good here.
My choice would be 'match'

It remebers me api for regexp. And it is their function for that.

Reply:
done